Scripts/Proxies/Vive/ViveProxy.cs

```diff
@@ -58,12 +60,12 @@ public override void Awake()
 #endif
         }
 
-        static void PostAnimate(ProxyHelper.ButtonObject[] buttons, Dictionary<Transform, ProxyAnimator.TransformInfo> transformInfos, ActionMapInput input)
+        static void PostAnimate(Affordance[] affordances, ProxyAffordanceMap.AffordanceDefinition[] affordanceDefinitions, Dictionary<Transform, ProxyAnimator.TransformInfo> transformInfos, ActionMapInput input)
         {
             var proxyInput = (ProxyAnimatorInput)input;
-
-            foreach (var button in buttons)
+            foreach (var button in affordances)
             {
+                var affordanceDefinition = affordanceDefinitions.Where(x => x.control == button.control).FirstOrDefault().animationDefinition;
```

Comment:
this also happens through ProcessInput (and thus per-frame), so let's avoid linq here